WMF Framework/CacheController.swift

```diff
@@ -10,7 +10,15 @@ public enum CacheControllerError: Error {
 
 public class CacheController {
     
+    //set for testing
+    public static var temporaryCacheURL: URL? = nil
```

Comment:
`CacheController`'s public interface gets a bit muddied in app targets if it's not more explicit that `temporaryCacheURL` is only for test targets. What do you think about using our `TEST` flag to ensure a path to it isn't possible in app targets (like https://github.com/wikimedia/wikipedia-ios/commit/47fed0b4e1c502b20199e82bb92a0beb2532c1d0)?

Reply:
Also done (good catch and I need to remember to use that macro more).